scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+                                        total_records += record_count
+                                        processed_prefectures += 1
+                                        
+                                        pbar.set_postfix({
+                                            'Records': f'{total_records:,}',
+                                            'Prefecture': prefecture[:10],
+                                            'Success': processed_prefectures
+                                        })
+                                    else:
+                                        failed_prefectures.append(pref_name)
+                                    
+                                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)